setup.cfg

```diff
@@ -40,6 +40,7 @@ testing =
 	# upstream
 	pytest >= 6
 	pytest-checkdocs >= 2.4
+	flake8 <= 4.0.1
```

Comment:
This was addressed in c64902b8cafa8062398ef173278a21b042b03a77 but only recently merged.
```suggestion
```